configure.ac

```diff
@@ -205,16 +205,16 @@ PHP_PROG_AWK
 PHP_PROG_BISON([3.0.0])
 PHP_PROG_RE2C([0.13.4])
 
+RE2C_FLAGS="--no-debug-info --no-generation-date"
```

Comment:
I don't think this `--no-debug-info` matches current behavior. At least I see `#line` annotations in the generated lexer right now.

Reply:
Yes, that is correct. The -i flag isn't used when building from Git with ./configure. It is used for release packages only. It was used also in PHP-7.3 btw...

Reply:
So, that's why this sync :)

Reply:
The sync should probably go in the other direction then, to not specify `-i` for release packages. Just because it's a release package doesn't mean that nobody is going to debug it (duh...)

Reply:
Is this problematic somehow that debug info (#line) is used in the release package? If not, then we can drop the -i from the genfiles script and simplify this completely, yes... But notice another thing here... The bison generated files. Probably I should merge this PR and the bison's PR into a single PR for better understanding. There are unnecessary *.output files.